README.md

```diff
@@ -1 +0,0 @@
-# learn-react
```

Comment:
Looks like you deleted the original README.md, leave this one alone

Reply:
i.e., put it back the way it was so there is no diff in your PR

Reply:
Ok, so the file is back, but you can see here that there's a diff.

The red shows what used to be in the file, and the green shows what's being changed by your PR